articles/overview/understanding-quantum-computing.md

```diff
@@ -0,0 +1,60 @@
+
+This is helpful to us, because in computing you can do lots of things with 1’s and 0’s. However, once a quantum particle has been measured and collapsed, it stays in that state forever and will always be a 1 or 0, and cannot be used for more calculations. As you’ll see later, though, in quantum computing there are operations that can “reset” a particle back to a superposition state so it can be useful again.
+
+### Entanglement
+
+Possibly the most interesting phenomenon of quantum physics is the ability of two or more quantum particles to become **entangled** with each other. When particles become entangled, they form a single system such that the quantum state of any one particle cannot be described independently of the quantum state of the other particles. This means that whatever operation or process you apply to one particle immediately takes effect on the other particles as well.
+
+In addition, particles can maintain this connection even when separated over incredibly large distances, even light-years. The effects of quantum measurement still apply to entangled particles, such that when one particle is measured and collapses, the other particle collapses as well. However, the second one always collapses in the *opposite* state as the first one – this particular property is very helpful in quantum computing. These unique properties of entangled particles can be used to transmit quantum information from one place to another by a process called **teleportation**, and may play a critical role in future cryptography applications.
+
+### Qubits and probability
+
+Classical computers store and process information in bits, which can have a state of either 1 or 0, but never both. The equivalent in quantum computing is the **qubit**, which represents the state of a quantum particle. Because of superposition, qubits can either be 1 or 0, or both, or anything in between. Depending on its configuration, a qubit has a certain *probability* of collapsing to 1 or 0, and this probabilistic state is where the power of quantum computing excels.
+
+In classical computing, uncertainty is not a good thing. In quantum computing, it’s built in.
+
+For example, with two bits in a classical computer, each bit can store 1 or 0, so together you can store four possible values – **00**, **01**, **10**, and **11** – but only one of those at a time. With two qubits in superposition, however, each qubit can store 1 or 0 or *both*, so you can represent the same four values simultaneously. Furthermore, when multiple entangled qubits are in superposition, they can process multiple options simultaneously. And with each qubit you add to the system, that power increases exponentially.
```

Comment:
It is not true in general that the power of a quantum computer increases exponentially with the number of qubits — rather, the difficulty of simulating a quantum computer using a classical computer increases exponentially. Comparing the power of each is significantly more subtle.